webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -263,6 +265,19 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
     setIsRunningDelete(experiment.state === RunState.Deleting);
   }, [experiment.state]);
 
+  const onClickContinueMultiTrialExp = useCallback(async () => {
+    try {
+      setIsRunningContinue(true);
+    await continueExperiment({
+      id: experiment.id,
+    });
+    const newPath = paths.experimentDetails(experiment.id);
+    routeToReactUrl(paths.reload(newPath));
+  } finally {
```

Comment:
no `catch` clause?

Reply:
probably worth notifying the user if the API fails to continue the experiment